### ISSUE TITLE
Add new "Remote" section, allows for following the local timeline of other instances

### ISSUE THREAD

**Issue by JackFromWisconsin** [mastodon/mastodon#21327]
### Pitch

The Remote section would be another timeline, but its posts would only be populated by a particular instance (chosen by user via search bar). This would allow for members of one instance view all posts on another instance in a feed, analogous to how hashtag searches work. (like hashtags, this may be limited to only posts that your instance knows about, not every single one.) 

The location of this may be another tab, (like how local and federated are separate), or may be integrated into the Explore section. 

### Motivation

The addition of this feature would allow for accounts to easily view local timelines of other instances while still being able to easily interact with the posts.

**Comment by JackFromWisconsin** [comment]
Similar issue:
#19948 (This request is for an ensemble Local timeline, whereas the Remote section would allow for viewing another instance's timeline)

**Comment by limdingwen** [comment]
Good suggestion. Related is Relays in Mastodon or the Bubble Timeline in Akkoma, but those are chosen by admins. But this seems to be a per-user choice, not admin's choice?

**Comment by JackFromWisconsin** [comment]
Yes this would be a per-user choice like how searching hashtags currently work.

**Comment by Timothyjchambers** [comment]
Deeply agree with a feature like this, akin to following Hashtags now. Would greatly reduce anxiety that new users have about choosing the "right" instance, and make for better discovery for all users who use this feature.

**Comment by nstbayless** [comment]
#3812 seems like a similar feature.

**Comment by nstbayless** [comment]
Also this: #20995

Seems like this is becoming one of the most commonly-requested features :)

**Comment by hkolbeck** [comment]
As a maintainer of a small instance I'd really like a feature similar to this, but I would argue that allowing users to subscribe to the local timeline of another instance would be setting up a trap. If a user of a small instance were to subscribe to the local timeline of mastodon.social or similar, it could blow up the maintainer's instance or hosting costs. I'd very much like something similar to Akkoma's bubbles

**Comment by JackFromWisconsin** [comment]
@hkolbeck I'm not sure how Akkoma's bubbles work. But as I stated in the original proposal: 
> (like hashtags, this may be limited to only posts that your instance knows about, not every single one.)

So unless you also followed every single mastodon.social person, then the remote section wouldn't show *everything*. This suggestion isn't about subscribing to another instance's local section, but rather introduce a new menu to view the local feed on your instance.

**Comment by hkolbeck** [comment]
@JackFromWisconsin Ahh, I see. Apologies for the misunderstanding.

**Comment by shevett** [comment]
This would be a phenomenal feature.  Given that instances are multiplying and many of them focus on a specific topic. and we all have only one account, but have multiple interests.  I'd love to be watch another servers local timelime on my own.
